### ISSUE TITLE
Focus keywords are wiped after upgrading to 4.0.0 and can't save new ones

### ISSUE THREAD

**Issue by MateuszEckardt** [ethercreative/seo#407]
### Description
Hello,

I would like to mention that after updating plugin to v4.0.0 all focus keywords are wiped and you cannot add new ones. When providing a new keyword in the field, on enter it saves the entry but the keyword is not saved. 

The only related issue I could find was [Keywords in headless mode](https://github.com/ethercreative/seo/issues/372) but I am not using Craft in headless mode, so I assume keywords should work.

I'd appreciate any feedback on this issue! Thanks

### Steps to reproduce

1. Create an entry OR edit an existing one
2. Go to your SEO field, Optimisation tab
3. Provide a keyword
4. Click enter
5. Entry saved
6. Go back to the SEO field, Optimisation tab
7. No keyword saved, empty field

### Additional info

- Craft version: 4.0.2 (pro)
- SEO version: 4.0.0
- PHP version: 8.1
- Database driver & version: MYSQL 5.7.38
- Other Plugins:

**Comment by rostockahoi** [comment]
I am experiencing the same. My best guess is that the entry form (maybe due to improved a11y?) submits when hitting enter before the keyword input can add the keyword.

There are 2 solutions here:
1. The keyword input prevents form submission if it is focused.
2. To create a keyword we use something different than the enter key.

I would prefer solution 1. Any idea how to do that?

**Comment by zadro** [comment]
Experiencing the same issue. However, keywords are still showing in source code.

**Comment by jakepm** [comment]
Experiencing the same issue. Can we get an acknowledgement on this at least, please?

**Comment by frederikvdbe** [comment]
Experiencing the same issue. Any updates on this please?

**Comment by elyselindsays** [comment]
I'm having the same issue. Getting "Entry Saved" when trying to add Keywords. Any ideas?

**Comment by defser** [comment]
Experiencing the same issue.

**Comment by darwinsoftware** [comment]
3 months later, no reply from the author, and 106 open issues for the plugin. Time to migrate to SEOmatic I think!

**Comment by alexjcollins** [comment]
We're taking a look into this one - I hope to come back to you all soon. Thanks for your patience.

**Comment by defser** [comment]
@alexjcollins Any news related this issue?

**Comment by kevinmu17** [comment]
There was an issue in the past when hitting enter it saved the entry, it's back again :)

The not saving part: Craft has a new way of auto-saving drafts, the submit is happening before the draft is saved so it won't save the keyword because that keyword didn't get "added" by then. 

IMO making multiple options to add is the best way here. 
- TAB key
- A physical button next to the input

**Comment by zadro** [comment]
Still no fix...is it time to abandon this SEO plugin?

**Comment by zadro** [comment]
Still no fix or update after 6 months...time to abandon this SEO plugin?

**Comment by darwinsoftware** [comment]
@alexjcollins, this bug is becoming really painful for us. On the one hand, you've created an excellent lightweight plugin that many of our clients like, but on the other hand, we've been waiting so long for a fix now that some of our clients are getting frustrated and we are feeling pressured to abandon the plugin.

This is a core piece of the UX for the plugin, and there isn't even a workaround.

**Comment by defser** [comment]
@darwinsoftware 

Relying on volunteers to maintain every open source project isn't long term sustainable. Funding open source projects could keep development moving. Maybe offer a small reward for a sollution or contribute yourself?

**Comment by rostockahoi** [comment]
Although you are certainly right @defser, this particular issue doesn't seem to be exactly easy to fix. That being said, it would be nice if we could at least get a hint on how to approach it.

Should we fix the input? For that we have to understand the implementation. I tried. (See my first comment)

Or should we replace the input with something else? Anyone knows a package or something that comes close to the current keyword input?

Finally, it would be nice if @alexjcollins could provide some insight if he could manage to invest more time into this issue if we provide a financial reward.

**Comment by darwinsoftware** [comment]
@defser, of course, I agree with what you're saying, and I respect and appreciate that this plugin exists as an open-source project that is adding a lot of value to the Craft community. 

That said, there has been no request for funding or help, and it seemed like this was in hand back in August. I would certainly be willing to contribute to a fund/reward to fix this bug, but I'd rather see a license fee added so that this valuable little plugin can justify improved support and maintenance.

**Comment by darwinsoftware** [comment]
> Finally, it would be nice if @alexjcollins could provide some insight if he could manage to invest more time into this issue if we provide a financial reward.

Willing to offer/contribute a fund to get this issue fixed. @alexjcollins ?
